core/src/main/scala/kafka/cluster/Partition.scala

```diff
@@ -306,17 +306,18 @@ class Partition(val topic: String,
       case Some(leaderReplica) =>
         // keep the current immutable replica list reference
         val curInSyncReplicas = inSyncReplicas
-        val numAcks = curInSyncReplicas.count(r => {
+
+        def numAcks = curInSyncReplicas.count { r =>
```

Comment:
@guozhangwang @junrao Is it correct that we don't use `numAcks` in this method outside of logging?